### PR TITLE
Added chickpea salad sandwich recipe

### PR DESCRIPTION
**Pull request by lukepfeiffer10**

### CODE REVIEW

File: content/chickpea-salad-sandwich.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Chickpea Salad Sandwiches"
+date: 2021-09-09T10:50:46-07:00
+draft: false
+serves: 4
+total_time_to_make: 15
+time_to_prepare: 15
+time_to_cook: 0
```
Comment (JayNewstrom) [inline content/chickpea-salad-sandwich.md:8]:
```suggestion
time_to_cook: 30
```

File: content/chickpea-salad-sandwich.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- 1/2 cup diced spicy pickles
+- 4 tsp fresh lemon juice
+- 1/2 tsp salt
+- Fresh pepper to taste
+- 1/4 cup vegan mayo
+- 1/4 teaspoon garlic powder
+{{% /ingredients %}}
+
+{{% instructions %}}
+- Drain chickpeas and place in a large bowl. Mash well using the bottom of a mason jar or cup.
```
Comment (JayNewstrom) [inline content/chickpea-salad-sandwich.md]: Maybe split this into 2 lines and remove the periods to make it consistent?

**Review by JayNewstrom — APPROVED** [review]

File: content/chickpea-salad-sandwich.md (hunk 1 above)
Comment (lukepfeiffer10) [reply content/chickpea-salad-sandwich.md:8]: Technically there is no cook time here unless `time_to_cook` is something else?

Comment (JayNewstrom) [reply content/chickpea-salad-sandwich.md:8]: You're right... I saw 2 * 15 which meant 30. The order here is what fooled me. This is correct as is.

**Pull request merged**